update_python_spec.lisp

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(ql:quickload :split-sequence)
 (rpcq::clear-messages)
 
+(defun -to_ (f)
+  "Replace all occurences of - (dash) to _ (underscore) in F."
+  (concatenate 'string
+               (loop :for s :across f
+                     :if (char= s #\-)
+                       :collect #\_
+                     :else
+                       :collect s)))
+
 (dolist (namespace '("messages" "core-messages"))
   (load (make-pathname :name namespace :type "lisp" :directory '(:relative "src")))
-  (with-open-file (f (make-pathname :name namespace :type "py" :directory '(:relative "rpcq"))
+  (with-open-file (f (make-pathname :name (-to_ namespace) :type "py" :directory '(:relative "rpcq"))
                      :direction ':output
                      :if-exists ':supersede
                      :if-does-not-exist ':create)
     (rpcq::python-message-spec f (gethash namespace rpcq::*messages*))
-    (format t "Wrote new ~A.py~%" namespace)))
+    (format t "Wrote new ~A.py~%" (pathname-name f))))
```

Comment:
can we do something like this:
```suggestion
    (format t "Wrote new ~S~%" f)))
```

Reply:
@stylewarning `Wrote new #<SB-SYS:FD-STREAM for "file /Users/mark.skilbeck/hackery/lisp/rpcq/rpcq/messages.py" {10035492A3}>.py`

Reply:
whoa no bueno. But if that's a stream, then why are we calling `pathname-name` on it?

Reply:
because it's a pathname

Reply:
your comment below resolves this